tool/plugin/lib/plugin.dart

```diff
@@ -533,39 +533,34 @@ class BuildCommand extends ProductCommand {
       }
 
       // TODO: Remove this when we no longer support AS 3.2.
-      File processedFile3, processedFile4;
-      String oldSource3, oldSource4, newSource;
+      var files = Map<File, String>();
```

Comment:
nit: write as `<File, String>{}`   instead of `Map<File, String>()`

Reply:
Done!